examples/huggingface/pytorch/text-generation/quantization/requirements_cpu_woq.txt

```diff
@@ -0,0 +1,17 @@
+accelerate
+datasets >= 2.0
+peft
+protobuf
+sentencepiece != 0.1.92
+--extra-index-url https://download.pytorch.org/whl/cpu
+torch==2.2.0+cpu
+transformers
+bitsandbytes  #baichuan
+transformers_stream_generator
+tiktoken  #qwen
+einops  #qwen
+git+https://github.com/intel/neural-speed.git@v1.0.1.dev0
```

Comment:
do we need the version for ns? 
I think we can follow the inc just 
git+https://github.com/intel/neural-speed.git

Reply:
improved.